ehrapy/plot/_sa_plot.py

```diff
@@ -0,0 +1,229 @@
+from __future__ import annotations
+
+from xmlrpc.client import Boolean
+
+import matplotlib.pyplot as plt
+import numpy as np
+from anndata import AnnData
+from lifelines import KaplanMeierFitter
+from matplotlib.axes import Axes
+from numpy import ndarray
+from statsmodels.regression.linear_model import RegressionResults
+
+import ehrapy as ep
+
+
+def ols(
+    adata: AnnData | None = None,
+    x: str | None = None,
+    y: str | None = None,
+    scatter_plot: Boolean | None = True,
+    ols_results: list[RegressionResults] | None = None,
+    ols_color: list[str] | None | None = None,
+    xlabel: str | None = None,
+    ylabel: str | None = None,
+    figsize: tuple[float, float] | None = None,
+    lines: list[tuple[ndarray | float, ndarray | float]] | None = None,
+    lines_color: list[str] | None | None = None,
+    lines_style: list[str] | None | None = None,
+    lines_label: list[str] | None | None = None,
+    xlim: tuple[float, float] | None = None,
+    ylim: tuple[float, float] | None = None,
+    show: bool | None = None,
+    ax: Axes | None = None,
+    **kwds,
+):
+    """Plots a Ordinary Least Squares (OLS) Model result, scatter plot, and line plot.
+
+    Args:
+        adata: :class:`~anndata.AnnData` object object containing all observations.
+        x: x coordinate, for scatter plotting
+        y: y coordinate, for scatter plotting
+        scatter_plot: If True, show scatter plot. Default is True
```

Comment:
I've always written defaults as

BLABLABLA (default: True)

Maybe for consistency you could adapt this?

Reply:
No problem.